packages/graphql-playground-react/src/components/Playground/DocExplorer/GraphDocs.tsx

```diff
@@ -224,43 +224,39 @@ class GraphDocs extends React.Component<
           onMouseDown={this.handleDocsResizeStart}
         />
         <div className="doc-explorer-gradient" />
-        {docsOpen && (
-          <div
-            className="doc-explorer"
-            onKeyDown={this.handleKeyDown}
-            onMouseMove={this.handleMouseMove}
-            tabIndex={0}
-            ref={this.setDocExplorerRef}
-          >
-            <div className="doc-explorer-container">
-              {emptySchema && <ColumnDoc>{emptySchema}</ColumnDoc>}
-              {!emptySchema &&
-                schema && (
-                  <RootColumn
-                    schema={schema}
-                    width={this.state.widthMap.root || columnWidth - 1}
-                    searchValue={this.state.searchValue}
-                    handleSearch={this.handleSearch}
-                    sessionId={this.props.sessionId}
-                  />
-                )}
-              {!emptySchema &&
-                navStack.map((stack, index) => (
-                  <ColumnDoc
-                    key={index}
-                    width={this.state.widthMap[stack.field.path] || columnWidth}
-                  >
-                    <FieldDoc
-                      schema={schema}
-                      field={stack.field}
-                      level={index + 1}
-                      sessionId={this.props.sessionId}
-                    />
-                  </ColumnDoc>
-                ))}
-            </div>
+        <div
+          className="doc-explorer"
+          onKeyDown={this.handleKeyDown}
+          onMouseMove={this.handleMouseMove}
+          tabIndex={0}
+          ref={this.setDocExplorerRef}
+        >
+          <div className="doc-explorer-container">
+            {emptySchema && <ColumnDoc>{emptySchema}</ColumnDoc>}
+            {schema && (
+              <RootColumn
+                schema={schema}
+                width={this.state.widthMap.root || columnWidth - 1}
+                searchValue={this.state.searchValue}
+                handleSearch={this.handleSearch}
+                sessionId={this.props.sessionId}
+              />
+            )}
+            {navStack.map((stack, index) => (
```

Comment:
Please add `!emptySchema` again, otherwise the column will incorrectly be rendered after deserialization, when there is not yet a schema